examples/_includes/base.xml.njk

```diff
@@ -3,14 +3,30 @@
   <screen>
     <styles>
       {% include 'styles.xml.njk' %}
-      {% block styles %}{% endblock %}
+      {% block styles %}
+        <style
+          id="Header__Switch"
+          alignItems="center"
+          flex="1"
+          flexDirection="row"
+          flexGrow="1"
+          justifyContent="flex-end"
+          columnGap="8"/>
+        <style id="Switch__Label" fontSize="14" fontWeight="bold"/>
+      {% endblock %}
     </styles>
     <body style="Body" safe-area="true">
       <header style="Header">
         {% if not hide_back_button %}
           <text action="back" href="#" style="Header__Back">Back</text>
         {% endif %}
         <text style="Header__Title">{{ title }}</text>
+        {% if show_hxml_toggle %}
+          <view style="Header__Switch">
+            <text style="Switch__Label">HXML Navigator</text>
+            <switch id="navigator-switch" xmlns:set-navigator-source="https://instawork.com/hyperview-set-navigator-source" trigger="change" action="set-navigator-source" set-navigator-source:source={% if hxml_toggle_enabled %} "external" {% else %} "hyperview" {% endif %} value={% if hxml_toggle_enabled %} "on" {% else %} "off" {% endif %}/>
```

Comment:
Adjusting NS / breaking up line / sorting attributes:
```suggestion
            <switch
              id="navigator-switch"
              trigger="change"
              action="set-navigator-source"
              xmlns:demo="https://hyperview.org/demo"
              demo:source="{% if hxml_toggle_enabled %}external{% else %}hyperview{% endif %}"
              value="{% if hxml_toggle_enabled %}on{% else %}off{% endif %}"
            />
```

additional question: could we rename `hxml_toggle_enabled` to `hv_navigator_enabled`, and rename the label to "Hyperview Navigator"?

Reply:
Fixed except the spacing as discussed